x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings.test.tsx

```diff
@@ -5,19 +5,32 @@
  * 2.0.
  */
 
+import '../../../__mocks__/shallow_useeffect.mock';
+
+import { setMockActions } from '../../../__mocks__';
+
 import React from 'react';
 
 import { shallow } from 'enzyme';
 
 import { ResultSettings } from './result_settings';
 
 describe('RelevanceTuning', () => {
+  const actions = {
+    initializeResultSettingsData: jest.fn(),
+  };
   beforeEach(() => {
+    setMockActions(actions);
     jest.clearAllMocks();
   });
 
   it('renders', () => {
     const wrapper = shallow(<ResultSettings engineBreadcrumb={['test']} />);
     expect(wrapper.isEmptyRender()).toBe(false);
```

Comment:
[optional] you could maybe check for `ResultSettingsTable` here now, since it's fairly important to the page

Reply:
Yes, great suggestion: 1a81febcc2c38da1e073e7d685acd450673da0a2